src/pages/_organizers.js

```diff
@@ -97,6 +97,20 @@ export default {
       colors: biStripes
     }
   ],
+  Paris: [
+    {
+      name: 'Paola Ducolin',
+      twitterLink: 'PolaDuco',
+      colors: []
+    }
+  ],
+  Lisbon: [
+    {
+      name: 'Paola Ducolin',
+      twitterLink: 'PolaDuco',
+      colors: []
+    }
+  ],
```

Comment:
Last comment - can we alphabetize these within the existing list? 😁 

Thank yooooouuuuu (cc @SaraVieira)

Reply:
Dooooone